src/extension/background-script/nostr/index.ts

```diff
@@ -36,6 +42,35 @@ class Nostr {
     event.sig = signature;
     return event;
   }
+
+  encrypt(pubkey: string, text: string) {
+    const key = secp256k1.getSharedSecret(this.getPrivateKey(), "02" + pubkey);
+    const normalizedKey = Buffer.from(key.slice(1, 33));
```

Comment:
what is this doing? 
should this be a compressed key? 
https://www.npmjs.com/package/@noble/secp256k1#getsharedsecretprivatekeya-publickeyb

```
isCompressed = false determines whether to return compact (33-byte), or full (65-byte) key
```